# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp
# Commit: d56484304ec3
# Full Hash: d56484304ec3abe3891ede9e8d141c95543d3e37
# Author: Ray Kraesig <rkraesig@mozilla.com>
# Date: 2023-09-16 09:14:45
# Regressor Bug: 1837079
# File Overlap Count: 2
# Description:
#   Bug 1837079 - [5/10] Create WinFileDialog IPDL protocol and implementation  r=gstoll,handyman,ipc-reviewers,nika
#   
#   Create and implement a new top-level IPC protocol, `PWinFileDialog`,
#   using the primitives from bug 1833450 and the new sandboxing type from
#   the previous commit.
# ==============================================================================

diff -r 36d3afd881b4 -r d56484304ec3 ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp
--- a/ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp	Fri Sep 15 18:10:00 2023 +0000
+++ b/ipc/glue/test/utility_process_xpcom/UtilityProcessTest.cpp	Fri Sep 15 18:10:01 2023 +0000
@@ -5,8 +5,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #if defined(ENABLE_TESTS)
+#  include "mozilla/ipc/UtilityProcessManager.h"
 #  include "mozilla/ipc/UtilityProcessTest.h"
-#  include "mozilla/ipc/UtilityProcessManager.h"
 #  include "mozilla/dom/Promise.h"
 #  include "mozilla/ProcInfo.h"
 #  include "mozilla/IntentionalCrash.h"